AdyenComponents/ACH Direct Debit/ACHDirectDebitComponent.swift

```diff
@@ -71,11 +71,11 @@ public final class ACHDirectDebitComponent: PaymentComponent,
     /// - Parameters:
     ///   - paymentMethod: The ACH Direct Debit payment method.
     ///   - context: The context object for this component.
-    ///   - configuration: Configuration for the component.
-    public convenience init(
+    ///   - configuration: ACHDirectDebitComponentConfiguration for the component.
```

Comment:
No need to change the type in the parameter description, it still is a "configuration" on paper.